.nvmrc

```diff
@@ -1 +1 @@
-lts/carbon
```

Comment:
Do we still need this file given we are setting the node version in `package.json`?

Reply:
The file is still here, just changed it to say `lts/erbium` instead

Reply:
Yeah - but my question is do we still need it at all in the codebase?

Reply:
Yes to ensure we all get the right version by doing `nvm use`